config/config.go

```diff
@@ -192,19 +192,54 @@ func (cfg *Config) migrateNetworks() bool {
 	var changes bool
 	for name, net := range cfg.Networks.All {
 		defaultCfg, knownDefault := Default.Networks.All[name]
-		oldChainContexts, knownOld := OldNetworks[name]
+		oldNetwork, knownOld := OldNetworks[name]
 		if !knownDefault || !knownOld {
 			continue
 		}
 
-		for _, oldChainContext := range oldChainContexts {
+		// Chain contexts.
+		for _, oldChainContext := range oldNetwork.ChainContexts {
 			if net.ChainContext == oldChainContext {
 				// If old chain context is known, replace with default.
 				net.ChainContext = defaultCfg.ChainContext
 				changes = true
 				break
 			}
 		}
+
+		// RPCs.
+		for _, oldRPC := range oldNetwork.RPCs {
+			if net.RPC == oldRPC {
+				// If old RPC is known, replace with default.
+				net.RPC = defaultCfg.RPC
+				changes = true
+				break
+			}
+		}
+
+		// Migrate consensus denominations for known paratimes.
+		for ptName, pt := range net.ParaTimes.All {
+			ptCfg, knownPt := defaultCfg.ParaTimes.All[ptName]
+			if !knownPt {
+				continue
+			}
+			if pt.ConsensusDenomination == ptCfg.ConsensusDenomination {
+				continue
+			}
+
+			pt.ConsensusDenomination = ptCfg.ConsensusDenomination
```

Comment:
In the migrated config file I notcied the consensus denomination is set to `_`:

```ini
[networks.mainnet.denomination]
decimals = 9
symbol = 'ROSE'

[networks.mainnet.paratimes.sapphire]
consensus_denomination = '_'
```

Is this expected?

Reply:
Yeah this is because the native denomination is represented by an empty string which is not handled correctly during config loading.

There should also be a section under paratime denominations named `_` representing the native denomination of the paratime.